Components/GameBoard/SquishableCardPanel.jsx

```diff
@@ -26,10 +26,6 @@ class SquishableCardPanel extends React.Component {
         let overflow = requiredWidth - overallDimensions.width;
         let offset = overflow / (handLength - 1);
 
-        if(!this.props.isMe) {
-            cards = [...this.props.cards].sort((a, b) => a.facedown && !b.facedown ? -1 : 1);
```

Comment:
Rather than removing it entirely, we should just add another condition so that it's sorted for hand (where revealing ordering is undesirable) but not for shadows (where ordering is needed). You could do that by either checking that `source` property is `'hand'`, or by adding a new boolean property that specifies it should be sorted / grouped (`groupVisibleCards` perhaps?)